src/server/config/schema.js

```diff
@@ -181,7 +181,8 @@ export default () => Joi.object({
   map: Joi.object({
     manifestServiceUrl: Joi.when('$dev', {
       is: true,
-      then: Joi.string().default('https://staging-dot-catalogue-dot-elastic-layer.appspot.com/v2/manifest'),
+      // then: Joi.string().default('https://staging-dot-catalogue-dot-elastic-layer.appspot.com/v2/manifest'),
```

Comment:
Is there a reason the commented out old version is still in the code?

Reply:
thx, good catch